arelle/Locale.py

```diff
@@ -12,6 +12,7 @@
 '''
 import sys, subprocess
```

Comment:
add `from __future__ import annotations` here

Reply:
this done